net/pdns/Makefile

```diff
@@ -0,0 +1,203 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pdns
+PKG_VERSION:=4.2.0-rc1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://downloads.powerdns.com/releases/
+PKG_HASH:=806362a6332139267c4fe888c717a35a5658438c894c6764f12be7184367e61e
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au>
+PKG_LICENCE:=GPL-2.0-only
+PKG_LICENCE_FILES:=COPYING
+
+PKG_INSTALL:=1
```

Comment:
Given how long it takes to compile, I recommend adding PKG_BUILD_PARALLEL:=1

Reply:
Added